CHANGELOG.md

```diff
@@ -1,38 +1,29 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
```

Comment:
When is the changelog to be updated now?  Is there still a manual change needed when merging new work? i.e. the developer should update changelog with unreleased changes before merging into `develop`? Or are we only updating the changelog now on version releases with `standard-version`?

Reply:
The latter - changelog will only be updated on version releases using `standard-version`